src/preamble.js

```diff
@@ -2153,16 +2153,28 @@ function integrateWasmJS(Module) {
 
   function getBinary() {
```

Comment:
We use it when we need synchronous access to it - for the interpreter, or when async compilation is off. Also, I think it's nice it's separate, conceptually.